lib/oauthenticator/version.rb

```diff
@@ -1,5 +1,5 @@
 # OAuthenticator 
 module OAuthenticator
   # OAuthenticator::VERSION
-  VERSION = "1.3.1"
+  VERSION = "1.3.4"
```

Comment:
actually that will mess up the pr. can you just push up the two commits for this pr?